src/main/java/com/taf/business/pages/DashboardsPage.java

```diff
@@ -0,0 +1,42 @@
+package com.taf.business.pages;
+
+import com.codeborne.selenide.Condition;
+
+import static com.codeborne.selenide.CollectionCondition.itemWithText;
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class DashboardsPage {
+    private static final String PAGE_HEADER = "//span[@title='All Dashboards']";
+    private static final String DASHBOARD_COLUMNS_NAME = "//span[@class='headerCell__title-short--3_s1A']";
+    private static final String DASHBOARD_ITEM_NAME = "//a[contains(@class, 'gridCell')]";
+    private static final String DASHBOARD_ITEM_DESCRIPTION =
+            "//div[@class='gridCell__grid-cell--3e2mS gridCell__align-left--2beIG dashboardTable__description--2TbtF']";
```

Comment:
is it possible to simplify locators? For example, replace this one with `//div[contains(@class, 'description')]`

Reply:
You are right. Thanks for the advice.